testing/python_lib/integration_base.py

```diff
@@ -145,7 +145,7 @@ def _get_faucet_config_path(self):
         return os.path.dirname(os.path.abspath(__file__)) + \
             (config_file_format % ('forch-faucet-1'))
 
-    def parallelize(self, target, target_args):
+    def parallelize(self, target, target_args, batch_size=None):
```

Comment:
seems like it might be better to default to 200?  Seems like the common case would be to limit things, and then if necessary people could explicitly make it bigger?

Reply:
Makes sense to me.